test/unit/rules/no-at-ember-render-modifiers-test.js

```diff
@@ -0,0 +1,73 @@
+import { MESSAGES } from '../../../lib/rules/no-at-ember-render-modifiers.js';
+import generateRuleTests from '../../helpers/rule-test-harness.js';
+
+generateRuleTests({
+  name: 'no-at-ember-render-modifiers',
+
+  config: true,
+
+  good: ['<div {{this.someModifier}}></div>', '<div {{someModifier}}></div>'],
+
+  bad: [
+    {
+      template: '<div {{did-insert}}></div>',
+      verifyResults(results) {
+        expect(results).toMatchInlineSnapshot(`
+          [
+            {
+              "column": 5,
+              "endColumn": 19,
+              "endLine": 1,
+              "filePath": "layout.hbs",
+              "line": 1,
+              "message": "${MESSAGES['did-insert']}",
```

Comment:
I'm not sure that snapshots should be using variables like this (presumably you manually added the variable here?). The point of snapshots is that they can be automatically updated by jest whenever anything in this object changes.

Reply:
Updated (thanks to a new lint rule!)